src/GitTools.IssueTrackers/Context/IssueTrackerContext.cs

```diff
@@ -1,29 +0,0 @@
-﻿namespace GitTools.IssueTrackers
-{
-    public class IssueTrackerContext : Disposable, IIssueTrackerContext
-    {
-        public IssueTrackerContext()
```

Comment:
To hold shared information (context) about an issue tracker, same as authentication. This way we don't have to inject zillion of parameters but keep everything nicely tied together.